src/Setup/WixBA/RootViewModel.cs

```diff
@@ -43,6 +43,7 @@ public RootViewModel()
         public ProgressViewModel ProgressViewModel { get; private set; }
         public UpdateViewModel UpdateViewModel { get; private set; }
         public IntPtr ViewWindowHandle { get; set; }
+        public Action CloseAction { get; set; }
```

Comment:
True it was removed in a subsequent commit.

On Mon, Aug 1, 2016 at 12:03 AM, Sean Hall <notifications@github.com> wrote:

> In src/Setup/WixBA/RootViewModel.cs
> <https://github.com/wixtoolset/wix4/pull/208#discussion_r72924466>:
>
> > @@ -43,6 +43,7 @@ public RootViewModel()
> >          public ProgressViewModel ProgressViewModel { get; private set; }
> >          public UpdateViewModel UpdateViewModel { get; private set; }
> >          public IntPtr ViewWindowHandle { get; set; }
> > +        public Action CloseAction { get; set; }
>
> I can't find where this is used.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/wixtoolset/wix4/pull/208/files/28721ab8e4985a5641b3efd7975a67cecd8f8d4a#r72924466>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHyCLdmpy9uDogfB6UDg0Tq-DL8B6xCWks5qbX4lgaJpZM4JVm8P>
> .
>